src/screens/Customers/EditCustomer.tsx

```diff
@@ -117,7 +126,6 @@ function EditCustomer(props: EditCustomerProps) {
 }
 
 const mapStateToProps = (state: RootState) => ({
-  currentCustomer: getCurrentCustomer() || EMPTY_CUSTOMER,
   currentSite: state.siteData.currentSite || EMPTY_SITE,
   user: state.userData.user || EMPTY_USER,
```

Comment:
I wrote a `selectCurrentUserId` selector for inventory data that you should use here